### PR TITLE
rfc: update the RFC process

### PR DESCRIPTION
**Pull request by vivekmenezes**

### CODE REVIEW

File: docs/RFCS/README.md [hunk 1]
```diff
@@ -110,25 +141,13 @@ conditions apply:
...
 
-7. Once the changes described in the RFC have been made, change the
+9. Once the changes described in the RFC have been made, change the
    status of the PR from `in-progress` to `completed`. If subsequent
    developments render an RFC obsolete, set its status to `obsolete`.
 
 When you mark a RFC as obsolete, ensure that its text references the
 other RFCs or PRs that make it obsolete.
 
-# Rationale about minimum delays and process complexity
```
Comment (knz) [inline docs/RFCS/README.md]: I think this section should stay to motivate the choices made in the rest of the process.

File: docs/RFCS/README.md [hunk 2]
```diff
@@ -14,7 +14,7 @@ What is a major change?
 A change is "major" and requires a RFC when any of the following
 conditions apply:
 
-- completing the change will cost you more than a week of work.
+- completing the change will cost you more than two weeks of work.
```
Comment (knz) [inline docs/RFCS/README.md]: This needs to be synchronized with the "how to contribute to cockroachdb" doc page, and also bounced back to Nate who was coordinating the definition of the contribution process.

**Review by knz — COMMENTED** [review]
I like these changes! The word "shepherd" is much better indeed.

File: docs/RFCS/README.md (hunk 1 above)
Comment (vivekmenezes) [reply docs/RFCS/README.md]: Leaving in with some modifications

**Comment by bdarnell** [comment]
:lgtm:

---

Review status: 0 of 1 files reviewed at latest revision, 1 unresolved discussion.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/cockroachdb/cockroach/20171#-:-L-bwU7VAMuOVYtCt0eZ:bnfp4nl)*

**Pull request merged**